x11/xorg-libxcb/Portfile

```diff
@@ -50,33 +51,28 @@ variant docs description "Install extra documentation" {
         DOXYGEN=${prefix}/bin/doxygen
 }
 
-variant python27 conflicts python34 python35 python36 python37 description {Use python 2.7} {
-    depends_run-append      port:python27
+variant python27 conflicts python35 python36 python37 description {Use python 2.7} {
+    depends_build-append    port:python27
```

Comment:
I'm almost wondering if we shouldn't just take this opportunity to remove the python27 variant. After all, python27 has less than six months to live before all python2 stuff is obsolete.

Reply:
That's also a good idea. I will remove python27 when I find time to update this PR.